plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -59,17 +63,23 @@ public void importOrderFile(Object importOrderFile) {
 		addStep(ImportOrderStep.createFromFile(getProject().file(importOrderFile)));
 	}
 
+	/** Use {@link #eclipse()} instead */
```

Comment:
> That would not be JavaDoc compliant.

Oh yes, you're right! Silly me. :wink:

I do understand that, from a Gradle user perspective, they'd write `eclipse([version]).configFile configFile.xml`, but I still think that, from a spotless-lib user perspective, it should be clearly stated which method _after_ `eclipse()` needs to be invoked to get the exact same behaviour as `eclipseFormatFile()`, so as to prevent confusion or ambiguity.

How about the following JavaDoc for `eclipseFormatFile(String, Object)`...
```
/**
  * Use {@link #eclipse(String) eclipse(String)}{@code .}{@link
  * EclipseConfig#configFile(Object...) configFile(Object...)} instead.
  */
```

...and the following for `eclipseFormatFile(Object)` instead?
```
/**
 * Use {@link #eclipse() eclipse()}{@code .}{@link EclipseConfig#configFile(Object...)
 * configFile(Object...)} instead.
 */
```